src/server/__tests__/server.test.js

```diff
@@ -458,11 +466,57 @@ describe('authentificaton', () => {
         .get(`/game/${matchID}/${endpoint}`)
         .set(
           'Authorization',
+          // missing 'Basic ' prefix
           Buffer.from(`0:${credentials[0]}`).toString('base64'),
         );
       expect(response.status).toBe(403);
     },
   );
+
+  it.each(endpoints)(
+    'is successful for correct credentials provided to %s',
+    async (endpoint) => {
+      // Try players
+
+      let response = await request(publicApiServer.callback())
+        .get(`/game/${matchID}/${endpoint}`)
+        .set(
+          'Authorization',
+          'Basic ' + Buffer.from(`0:${credentials[0]}`).toString('base64'),
+        );
+      expect(response.status).not.toBe(403);
+    },
+  );
+
+  it.each(endpoints)(
+    'is successful for correct spectator credentials provided to %s',
```

Comment:
is the Space before "is" provided for by the variable?

Reply:
I am not sure I get your point. I think the space in front of "is" is provided by the testing framework?

Reply:
It’s usually even displayed on a separate line and indented, isn’t it?

Reply:
Seems fine to me then :)